OptiFineDoc/assets/minecraft/optifine/lang/pt_br.lang

```diff
@@ -12,40 +12,40 @@ of.general.smart=Inteligentes
 of.key.zoom=Zoom
 
 # Message
-of.message.aa.shaders1=O anti-aliasing não é compatível com o Shaders.
-of.message.aa.shaders2=Desative o Shaders para ativar essa opção.
+of.message.aa.shaders1=O anti-aliasing não é compatível com o Shader.
```

Comment:
Bear in mind that the plural form is correct in the original locale as proper noun. Keep the _-s_ since it's a name of a graphic feature.

Reply:
"com shaders" instead of "com o shader/shaders" maybe?